mlx/coverity/coverity.py

```diff
@@ -150,30 +153,44 @@ def get_filtered_defects(self, node):
                     "rows": [list of dictionaries {"key": <key>, "value": <value>}]
                 }
         """
-        report_info("obtaining defects... ")
+        LOGGER.info("obtaining defects... ")
         column_names = set(node["col"])
         if "chart_attribute" in node and node["chart_attribute"].upper() in node.column_map:
             column_names.add(node["chart_attribute"])
         defects = self.coverity_service.get_defects(self.stream, node["filters"], column_names, self.snapshot)
-        report_info("%d received" % (defects["totalRows"]))
+        LOGGER.info("%d received" % (defects["totalRows"]))
         return defects
 
 
+def validate_coverity_credentials(app):
+    """Validate the configuration of coverity_credentials.
+
+    Args:
+        app (sphinx.application.Sphinx): Sphinx' application object.
+    """
+    for required_element in ["hostname", "username", "password", "stream"]:
+        if required_element not in app.config.coverity_credentials:
+            LOGGER.error(f"{required_element} is a required configuration in 'coverity_credentials' in conf.py")
```

Comment:
`set.difference` with the 4 required elements as the source would be cleaner and faster and only result in 1 error log.

Reply:
Resolved in a43c680ebce09b0b1507ff525b54b982ee1b1566